src/main/java/haste/tasks/Event.java

```diff
@@ -0,0 +1,35 @@
+package haste.tasks;
+
+import haste.commands.Parser;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents an Event.
+ */
+public class Event extends Task {
+    protected LocalDateTime start;
+    protected LocalDateTime end;
+
+    /**
+     *
+     * @param description Task description.
+     * @param start Start time of the Task.
+     * @param end End time of the Task.
+     * @param isComplete Completion status of Task.
+     */
+    public Event(String description, LocalDateTime start, LocalDateTime end, boolean isComplete) {
+        super(description,isComplete);
```

Comment:
White space within a statement - Java reserved words should be followed by a white space.